src/Build/Logging/SerialConsoleLogger.cs

```diff
@@ -535,10 +612,17 @@ public override void MessageHandler(object sender, BuildMessageEventArgs e)
 
                 string nonNullMessage = null;
 
+                //9 times out of targe the framework will be left as null but functionality incase future changes
```

Comment:
?

Reply:
Oops sorry that was  a note for myself that I forgot to delete